CHANGELOG.md

```diff
@@ -1,231 +1,334 @@
-## 0.10.0 / 2015-06-24
+# Change Log
 
-### Bug Fixes
+## [1.0.0.beta.2](https://github.com/test-kitchen/kitchen-ec2/tree/1.0.0.beta.2) (2016-02-26)
```

Comment:
Are you going to do the final `1.0.0` changelog after this is merged, or should that be in this PR?